ec2, ecs: EOL Amazon Elastic Inference (#42137)
hashicorp/terraform-provider-aws: website/docs/guides/version-6-upgrade.html.markdown

`tags_all` has been removed.

## data-source/aws_ecs_task_definition

> Remove `inference_accelerator` from your configuration—it no longer exists. Amazon Elastic Inference reached end of life in April 2024.

Reviewer: Perhaps rephrase slightly to align with the other existing removal entries?

```suggestion
`inference_accelerator` has been removed. Amazon Elastic Inference reached end of life in April 2024.
```

Author: I think the guide should be telling people what they need to do, imperative language: "remove this, do that, update this"

Reviewer: Seems reasonable to me, though most entries are not currently doing that. Perhaps we just need to plan for standardizing at the end so each section is consistent.
